app/styles/app.scss

```diff
@@ -595,6 +596,9 @@ hr {
             }
         }
     }
+    button {
```

Comment:
This seems like a very general style, that would affect more than just your component. Something to watch out for.